e2e/snapshot.go

```diff
@@ -6,8 +6,8 @@ import (
 	"strings"
 	"time"
 
-	snapapi "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
-	snapclient "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/clientset/versioned"
+	snapclient "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
```

Comment:
you changed other places to `snapapi`, better to use the same alias everywhere?

Reply:
@nixpanic I dont get above comment, in e2e files , its `snapapi` everywhere. did I miss any?

Reply:
You updated to use
```
snapapi "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
```
 everywhere else. Here you rename the alias `snapapi` to `snapclient`. Better use `snapapi` here as it used to be, and what you used in other locations.

Reply:
ok. will have the same notion here too.. changed in the last refresh